packages/example/src/pages/components/markdown.mdx

```diff
@@ -145,7 +145,7 @@ recommend using `h2` tags for section headings within your content.
 [I’m a markdown link with title](https://www.carbondesignsystem.com 'Carbon’s Homepage')
 
 URLs and URLs in angle brackets will automatically get turned into links.
-http://www.example.com or <http://www.example.com> and sometimes example.com
+http://www.example.com or \<http://www.example.com> and sometimes example.com
```

Comment:
```suggestion
`http://www.example.com` or `<http://www.example.com>` and sometimes example.com
```

Reply:
Lets just add these as code blocks so they are not interpreted, as this is technically what it should look in markdown code anyways.